pkg/plugin/get.go

```diff
@@ -154,7 +154,7 @@ func (o *GetOptions) Run() error {
 
 func newGetTable(out io.Writer) *tablewriter.Table {
 	table := tablewriter.NewWriter(out)
-	table.SetHeader([]string{"Namespace", "Name", "Desired", "Current", "Ready", "Up-to-date", "Available", "Ignored Unresponsive Nodes", "Status", "Active RS", "Canary RS", "Canary Paused", "Age"})
+	table.SetHeader([]string{"Namespace", "Name", "Desired", "Current", "Ready", "Up-to-date", "Available", "Ignored Unresponsive Nodes", "Status", "Reason", "Active RS", "Canary RS", "Age"})
```

Comment:
This change needs to be reflected in the CRD as well: https://github.com/DataDog/extendeddaemonset/blob/master/pkg/apis/datadoghq/v1alpha1/extendeddaemonset_types.go#L126-L136